### PR TITLE
Home mobile

### PR DESCRIPTION
**Pull request by jeffreyzarate**

### CODE REVIEW

**Review by BrandonSlade — APPROVED** [review]
Everything looks fine.

File: src/app/project/view-projects-logic/view-projects-logic.component.scss [hunk 1]
```diff
@@ -145,3 +145,115 @@
...
+    width: 80px;
+    display: inline-block;
+        font-weight: bold;
+  }
+
+  .mat-header-row {
+    display: none;
+  }
+
+  // tbody {
```
Comment (mgoshorn) [inline src/app/project/view-projects-logic/view-projects-logic.component.scss]: Remove commented out rules

File: src/app/project/view-projects-logic/view-projects-logic.component.scss [hunk 2]
```diff
@@ -145,3 +145,115 @@
...
+  display: none;
+} 
+
+@media screen and (max-width: 960px) {
+  .mat-elevation-z8 {
+    background: transparent;
+    box-shadow: none;
+  }
+
+  #project-image
```
Comment (mgoshorn) [inline src/app/project/view-projects-logic/view-projects-logic.component.scss:161]: Keep formatting of rules consistent

File: src/app/project/view-users-projects/view-users-projects.component.scss [hunk 3]
```diff
@@ -126,4 +126,116 @@
...
+      width: 80px;
+      display: inline-block;
+          font-weight: bold;
+    }
+  
+    .mat-header-row {
+      display: none;
+    }
+  
+    // tbody {
```
Comment (mgoshorn) [inline src/app/project/view-users-projects/view-users-projects.component.scss]: Remove commented out rules

**Review by mgoshorn — APPROVED** [review]

**Review by mgoshorn — CHANGES_REQUESTED** [review]
@jeffreyzarate See above notes

**Review by mgoshorn — APPROVED** [review]

**Pull request merged**